apps/src/lib/kits/maker/ui/EligibilityChecklist.jsx

```diff
@@ -144,11 +153,11 @@ export default class EligibilityChecklist extends Component {
             According to our data, your school has fewer than 50% of students that are
             eligible for free/reduced-price lunches. This means that we can bring down
             the cost of the $325 kit to just $97.50.{" "}
-            <b>
```

Comment:
I agree with removing {b} but would it be possible to use {strong} rather than {div} here? Screenreaders are able to emphasize the strong tag, so it become perceivable by folks using screenreaders.

Reply:
fwiw, the reason i removed the b is because for whatever reason, just bolding gotham 4r seemed to show very little difference in bold. i saw that we use gotham 7r where we want bold elsewhere, so switched to that.
will add a strong. thx for the pointer :)